lectures/deep.js

```diff
@@ -497,6 +498,11 @@ add(slide('Long Short Term Memory (LSTM)',
   //parentCenter(xtable('$\\Encode(h_{t-1}, x_t) = \\sigma($', vectorBox('$V$', d, d, 'blue'), vectorBox('$h_{t-1}$', d), '$+$', vectorBox('$W$', d, nx = 8, 'blue'), vectorBox('$x_t$', nx), '$) =$', vectorBox('$h_t$', d)).center()),
 _).leftHeader('[Schmidhuber &amp; Hochreiter, 1997]'));
 
+add(slide('LSTM Intuition',
+  parentCenter(image('images/lstm-intuition.png').width(700)),
```

Comment:
I don't think we should spend much time on LSTMs, especially since Transformers are more dominant these days.  It's pretty arcane, and in this lecture, all we can really do is give a broad overview of the high-level ideas.

Reply:
Good point. In that case I think better to just give the intuitive slide, since you wanted to rework discussion of vanishing gradients anyway